File: management_ms/src/main/java/video/to/image/management_ms/infra/adapters/outbound/messaging/events/ProcessVideoEvent.java
package video.to.image.management_ms.infra.adapters.outbound.messaging.events;

import java.util.UUID;

public record ProcessVideoEvent(
        UUID videoProcessId,
        UUID userId,
        String bucket,
        String storageKey,
        String outputZipKey,
        String originalFileName,
        String contentType
) {
}

File: worker_ms/src/main/kotlin/video/to/image/worker_ms/messaging/ProcessVideoEvent.kt
package video.to.image.worker_ms.messaging

import java.util.UUID

data class ProcessVideoEvent(
    val videoProcessId: UUID,
    val userId: UUID,
    val bucket: String,
    val storageKey: String,
    val outputZipKey: String,
    val originalFileName: String,
    val contentType: String,
)
